packages/snap/src/keyring.ts

```diff
@@ -741,7 +741,7 @@ export class AccountAbstractionKeyring implements Keyring {
     const transaction =
       transactions[0] ?? throwError(`[Snap] Transaction is required`);
     logger.info(
-      `[Snap] PrepareUserOp for transaction\n: ${JSON.stringify(
+      `[Snap] PrepareUserOp for transaction prepareUserOperation\n: ${JSON.stringify(
```

Comment:
if we can let's avoid this snap change, since this will change the snap hash (snap.manifest.json)

Reply:
hmm sounds good will revert this!